lint381/c.py

```diff
@@ -11,6 +11,20 @@
 
 
 @linter.register
+@with_matched_tokens(start=match_regex("^sizeof$"), end=match_regex("^char$"))
+def prohibited_unsigned_char(source, *, match):
+    """Flag sizeof(char) as it defined to be 1 and is redundant."""
+    error_message = "sizeof(char) should not be used as it is defined by the "\
```

Comment:
I think the error message is too wordy compared to the other error messages. For example, compare with

  * Don't cast the result of 'malloc'
  * Enum member 'foo' should be all-caps

Maybe:

  * Don't use 'sizeof(char)'
  * 'sizeof(char)' is redundant as it is always 1

Or something else of your suggestion.